entry.go

```diff
@@ -122,7 +122,7 @@ func (entry Entry) log(level Level, msg string) {
 	// panic() to use in Entry#Panic(), we avoid the allocation by checking
 	// directly here.
 	if level <= PanicLevel {
-		panic(&entry)
```

Comment:
@mkorolyov why did you do this? The problem isn't in logrus library, `panic` and `recover` accepts and returns just interface. With this "fix" you lose any additional information that can goes with message.

To fix original problem it's required to fix place where recover is called.